test/support/log.js

```diff
@@ -0,0 +1,17 @@
+
+import moment from 'moment';
+import util from 'util';
+
+import {
+  config
+} from './index';
```

Comment:
I strongly object to having the separate `utils` directory.  Let's use `support` for things that are not page objects and `support/pages` for page objects.  If we go the route of npm link, or sync, or some other technique to share the functional automation with another project, having a separate (but required) utils directory just makes it more painful.